### PR TITLE
HPCC-27924 PT-BR Translations for 8.8.X

### PR DESCRIPTION
**Pull request by kunalaswani**
Translations added for Brazilian Portuguese (PT-BR).

Signed-off-by: Kunal Aswani <kunal.aswani@lexisnexis.com>

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [x] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [x] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [x] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [x] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [x] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

File: esp/src/src/nls/pt-br/hpcc.ts [hunk 1]
```diff
@@ -376,6 +388,7 @@
     IsLibrary: "Biblioteca?",
     IsReplicated: "É Replica?",
     IssueReporting: "Reportagem de Problemas",
+    JobID: "'ID de trabalho'",
```
Comment (dehilsterlexis) [inline esp/src/src/nls/pt-br/hpcc.ts:391]: single quote (my bad)

File: esp/src/src/nls/pt-br/hpcc.ts [hunk 2]
```diff
@@ -423,8 +436,9 @@
     LocalFileSystemsOnly: "Somente Sistemas de Arquivos Locais",
     Location: "Localização",
     Lock: "Bloquear",
-    Log: "Registro",
     log_analysis_1: "log_analysis_1*",
+    Log: "Registro",
+    LogAccessType: "'Tipo de acesso ao registro'",
```
Comment (dehilsterlexis) [inline esp/src/src/nls/pt-br/hpcc.ts:441]: single quote (my bad)

File: esp/src/src/nls/pt-br/hpcc.ts [hunk 3]
```diff
@@ -455,6 +469,7 @@
     Mappings: "Mapeamento",
     Mask: "Mask (bloqueador de caracteres)",
     Max: "Max",
+    MaxConnections: "'Conexões máximas'",
```
Comment (dehilsterlexis) [inline esp/src/src/nls/pt-br/hpcc.ts:472]: single quote (my bad)

File: esp/src/src/nls/pt-br/hpcc.ts [hunk 4]
```diff
@@ -558,6 +576,7 @@
     PackageMap: "Mapa de Pacotes",
     PackageMaps: "Mapas do Pacote de Dados",
     PackagesNoQuery: "Pacotes sem queries",
+    PageSize: "'Tamanho da Página'",
```
Comment (dehilsterlexis) [inline esp/src/src/nls/pt-br/hpcc.ts:579]: single quote (my bad)

File: esp/src/src/nls/pt-br/hpcc.ts [hunk 5]
```diff
@@ -735,6 +757,7 @@
     SelectValue: "Selecione um valor",
     SendEmail: "Enviar email",
     Separators: "Seperadores",
+    Sequence: "'Seqüência'",
```
Comment (dehilsterlexis) [inline esp/src/src/nls/pt-br/hpcc.ts:760]: single quote (my bad)

File: esp/src/src/nls/pt-br/hpcc.ts [hunk 6]
```diff
@@ -824,13 +847,15 @@
     Targets: "Alvos",
     TargetScope: "Escopo de Destino",
     TargetWuid: "Unidade de trabalho alvo",
+    TechPreview: "'Visualização Técnica'",
```
Comment (dehilsterlexis) [inline esp/src/src/nls/pt-br/hpcc.ts:850]: single quote (my bad)

File: esp/src/src/nls/pt-br/hpcc.ts [hunk 7]
```diff
@@ -824,13 +847,15 @@
...
     Terminators: "Terminadores",
     TestPages: "Páginas de Teste",
     Text: "Texto",
     TheReturnedResults: "Os Resultados Devolvidos",
     ThorMasterAddress: "Endereço de metre thor",
     ThorNetworkAddress: "Endereço de Rede do Thor",
     ThorProcess: "Processo de Thor",
+    ThreadID: "'ID do Tópico'",
```
Comment (dehilsterlexis) [inline esp/src/src/nls/pt-br/hpcc.ts:858]: single quote (my bad)

File: esp/src/src/nls/pt-br/hpcc.ts [hunk 8]
```diff
@@ -124,10 +127,14 @@
     ConfirmPassword: "Confirmar Senha",
     ConfirmRemoval: "Tem certeza que quer fazer isso?",
     ContactAdmin: "Se você deseja renomear este grupo, por favor, entre em contato com o administrador do LDAP",
+    Container: "Pacote",
+    ContainerName: "Nome do Pacote",
     Content: "Conteúdo",
     Contents: "Conteúdo",
     ContentType: "Tipo de Conteúdo",
     ContinueWorking: "Contínua Trabalhando",
+    CookiesAcceptButtonText: "'Permitir Cookies'",
```
Comment (dehilsterlexis) [inline esp/src/src/nls/pt-br/hpcc.ts:136]: single quote (my bad)

File: esp/src/src/nls/pt-br/hpcc.ts [hunk 9]
```diff
@@ -124,10 +127,14 @@
...
+    CookiesNoticeLinkText: "'Nossa notificação de cookies'",
```
Comment (dehilsterlexis) [inline esp/src/src/nls/pt-br/hpcc.ts:137]: single quote (my bad)

File: esp/src/src/nls/pt-br/hpcc.ts [hunk 10]
```diff
@@ -290,6 +299,7 @@
     File: "Arquivo",
     FileAccessCost: "Custo de Acesso ao Arquivo",
     FileCluster: "Arquivos Cluster",
+    FileCostAtRest: "'Custo do Arquivo em Repouso'",
```
Comment (dehilsterlexis) [inline esp/src/src/nls/pt-br/hpcc.ts:302]: single quote (my bad)

File: esp/src/src/nls/pt-br/hpcc.ts [hunk 11]
```diff
@@ -361,6 +371,8 @@
     Inactive: "Inativo",
     IncludeSlaveLogs: "Incluie logs escravos",
     Index: "Índice",
+    Indexes: "'Índices'",
```
Comment (dehilsterlexis) [inline esp/src/src/nls/pt-br/hpcc.ts:374]: single quote (my bad)

File: esp/src/src/nls/pt-br/hpcc.ts [hunk 12]
```diff
@@ -361,6 +371,8 @@
...
+    IndexesOnly: "'Somente Índices'",
```
Comment (dehilsterlexis) [inline esp/src/src/nls/pt-br/hpcc.ts:375]: single quote (my bad)

**Review by dehilsterlexis — COMMENTED** [review]
Kunal, stupidly I got some single quotes into the text and didn't see them when I cut and pasted them. Sorry about that.

**Comment by kunalaswani** [comment]
@dehilsterlexis should be correct now, please check

**Review by dehilsterlexis — APPROVED** [review]
Look good

**Review by richardkchapman — APPROVED** [review]

**Pull request merged**